src/utils/getIsUnified.js

```diff
@@ -0,0 +1,36 @@
+// Recidiviz - a data platform for criminal justice reform
+// Copyright (C) 2021 Recidiviz, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// =============================================================================
+const getIsUnified = (stateCode) => {
+  switch (stateCode) {
+    case "US_AK":
+      return true;
+    case "US_CT":
+      return true;
+    case "US_DE":
+      return true;
+    case "US_HI":
+      return true;
+    case "US_RI":
+      return true;
+    case "US_VT":
+      return true;
```

Comment:
I think you could have all these cases fall through to a single `return true` statement? might be a little easier to read